### DISCUSSION TITLE
feign traceId

### DISCUSSION THREAD

**Discussion by xiebin2015** [apache/skywalking#8274] [Q&A]
Hello, I'm using version 8.9. For a spring cloud based project, there is a request to call another service through feign, but the tracking IDS obtained by the two services are inconsistent. Why？

**Comment by wu-sheng** [comment]
This is how we tested Feign. You could check on your own.

https://github.com/apache/skywalking-java/tree/main/test/plugin/scenarios/feign-scenario

**Reply by xiebin2015** [reply]
I deployed skywalking according to the tutorial. The micro service adopts the springcloud architecture and the thread isolation strategy between services. However, the tracking ID of the call chain printed in the log is different. It is disconnected. I don't know why. I hope I can ask for help

**Reply by wu-sheng** [reply]
Plugins are contributed by global people. What we did is checking the test cases(like I provided to you about Feign), and if tests passed, then we think this plugin is good for some cases. We can't afford to debug for every one and every case.

**Reply by wu-sheng** [reply]
So, as a user of open source project, it is always better you could debug on this, and check what is going on.

**Reply by xiebin2015** [reply]
The problem has been solved. The original is that the hystrixcurrencystrategy has been rewritten before, resulting in the skywalking plugin not taking effect. After adjustment, the problem has been solved. Thank you

**Reply by marker-wu** [reply]
> The problem has been solved. The original is that the hystrixcurrencystrategy has been rewritten before, resulting in the skywalking plugin not taking effect. After adjustment, the problem has been solved. Thank you

@xiebin2015 May I ask how to solve this problem? I also encountered the same problem. I customized the concurrency strategy of Hystrix, which caused the traceID of the two services to be inconsistent.
